### ISSUE TITLE
First song (nr. "00") in a music set/programme is always skipped

### ISSUE THREAD

**Issue by Vindigo89** [OpenTTD/OpenTTD#8946]
## Expected result
All songs in a music set are played when choosing to play "all"

## Actual result
The firste song, marked nr. "00" is always skipped, in a music set, unless it is selected in a "custom" set

## Steps to reproduce
Start any music set? The song numbered "00" is always shown in the left-hand "track index" panel, but not included in the right-hand "programme - all" panel. This is also obvious due to one less song being included in the latter panel.

Some added context: I have downloaded several music sets in-game, but never added any of my own music etc.
This isn't a huge issue, of course, but seems like a weird bug that should be fairly easy to fix maybe?

**Comment by 2TallTyler** [comment]
Number 00 is played on the main menu screen. I don't see why it shouldn't play in-game though.

**Comment by glx22** [comment]
It's done intentionally https://github.com/OpenTTD/OpenTTD/blob/e8a94dc8bbffbf3a48c9826531e116af5ff31396/src/music_gui.cpp#L109-L124
But of course you can create a custom playlist with all songs, including "00".

**Comment by Vindigo89** [comment]
Thanks for the help! For some weird reason, I got the impression that there was a limit to the number of songs one could include in a "custom" playlist, but when I checked again, there quite clearly wasn't. I'll just use that as a workaround.

I would note that, as referenced by 2TallTyler above, it might make sense for the original music set, since it skips the "main menu song" by default, but for all the additional, download, music sets, it ends up skipping one the songs. I'm not really able to make heads or tales of the code you pasted, glx22, so this may be redundant, but having the option or setting of including song nr. 00 for all music sets _except_ the original one seems like a good idea to me. Took me a while to notice I had skipped one song in the ten or so music sets I'd been listening to thus far.
